tests/Place/Canonical/CanonicalServiceTest.php

```diff
@@ -203,7 +203,7 @@ public function setUp(): void
      */
     public function it_will_return_the_MPM_place(): void
     {
-        $canonicalId = $this->canonicalService->getCanonical(1);
+        $canonicalId = $this->canonicalService->getCanonical('1');
```

Comment:
To avoid confusion I would sure, that everyone sees it's a `string`, so maybe also `cluster_1` like you did in `DBALDuplicatePlaceRepositoryTest.php` below.